File: JAVA/Program18.java
/*

Accept number from user and display below pattern.
Input : 5
Output : A B C D E

*/

import java.util.*;

class Demo18
{
	 public void Display(int iNo)
	 {
	 	int i = 0;
	 	char ch = 'A';

         System.out.print("\n");

	 	for(i = 1; i<= iNo; i++)
	 	{
	 		System.out.print(ch+"\t");
	 		ch++;
	 	}
	 }

}

class Program18
{
	public static void main(String[] args) 
	{
	   Scanner sobj = new Scanner(System.in);
	   Demo18 dobj = new Demo18();

	   int iVal = 0;
       
       System.out.print("\nEnter Number : ");
       iVal = sobj.nextInt();

	   dobj.Display(iVal);

	   System.out.print("\n");

	}
}

File: C#/Program18.cs
/*
Write a program which accept number from user and return difference between 
summation of all its factors and non factors.
Input : 12
Output : -34 (16 - 50)
*/


using System;

class Demo{
  public int DifferenceOfFactOrNonFactors(int iNo){
    int iFact = 0;
    int iNonFact = 0;

    for(int i = 1; i<iNo; i++)
    {
        if(iNo % i == 0)
        iFact = iFact+i;
        else
        iNonFact = iNonFact+i;
    }

    return iFact-iNonFact;
}
}

class Program18{

    public static void Main(string[] args){

        int iNo = Convert.ToInt32(Console.ReadLine());
        Demo dobj = new Demo();

        int iRet = dobj.DifferenceOfFactOrNonFactors(iNo);

        Console.WriteLine(iRet); 
    }
}


